polybar-scripts/info-optimus-manager/README.md

```diff
@@ -0,0 +1,24 @@
+# polybar-optimus-manager
+
+A script that shows the current GPU in use, given that the GPUs are managed by [optimus-manager](https://github.com/Askannz/optimus-manager). The script also allows for switching GPU in one click, just **please** be aware that this will log you out of your session without asking for confirmation. See below how this can be enabled.
+
+![polybar-optimus-manager](screenshots/1.png)
+
+## Configuration
+
+1. Make sure you have `enable-ipc=true` in the polybar config file.
+2. Edit the script and change the "Configuration" part to your liking. This is not necessary if you don't use the "Hybrid" mode in optimus-manager and you're happy with the default outputs. 
+3. (Optional, enables switching GPU when the module is left-clicked). Uncomment the line `;click-left = polybar-msg hook info-optimus-manager 2` in the module.
+
+
+## Module
+
+```ini
+[module/info-optimus-manager]
+type = custom/ipc
```

Comment:
Is there a reason why you use ipc and not a click event?

Reply:
Yes, originally I wanted to have an additional feature where if you clicked once the module would ask you to confirm, with a timer ticking to the left of the text. I thought it would be appropriate to use ipc for this. This was however scrapt. 

The reason I kept it was because it's unnecessary for Polybar to continually poll for data (i.e run the script at a given interval), since the module is for a powersaving feature in laptops and since the gpu only ever changes in between sessions. 

However, it's indeed probably better to just use the common `custom/script` type and set it to execute at a rather high interval. Readme will be updated with the next commit.